CMakeLists.txt

```diff
@@ -407,6 +407,10 @@ endif()
 
 find_program(LUACHECK_PRG luacheck)
 
+if(NOT GPERF_PRG)
```

Comment:
The documentation says it will be set to "<VAR>-NOTFOUND" but I will try and see what happens...